tests/tests/test-stake.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from "chai";
+import { step } from "mocha-steps";
+
+import { describeWithMoonbeam, createAndFinalizeBlock } from "./util";
+
+describeWithMoonbeam("Moonbeam RPC (Stake)", `simple-specs.json`, (context) => {
+  const GENESIS_ACCOUNT = "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b";
+  const GENESIS_STAKED = "100000";
+  const ACCOUNT_BALANCE = "340282366920938463463374607431768111455";
+  step("validator bond reserved in genesis", async function () {
+    const account = await context.polkadotApi.query.system.account(GENESIS_ACCOUNT);
+    expect(account.data.reserved.toString()).to.equal(GENESIS_STAKED);
+  });
+
+  step("validator set in genesis", async function () {
+    const validators = await context.polkadotApi.query.stake.validators();
+    expect((validators[0] as Buffer).toString("hex").toLowerCase()).equal(GENESIS_ACCOUNT);
+  });
+
+  step("issuance minted to the sole validator for authoring blocks", async function () {
```

Comment:
so 49 is the amount mined for the first 40 blocks and 49 also the amount mined for blocks 40-60?

Reply:
@4meta5 to confirm

Reply:
The first payment is for blocks 1-20, the second payment is for blocks 21-40. Both payments are delayed by 2 rounds so that is why we check for the first payment after the first 40 blocks and then the second payment after block 60.

Yes, `IssuancePerRound` is 49 so 49 is minted every 20 blocks (Round), but the payout itself is delayed by 2 rounds (`BondDuration`).

Reply:
ok good to know 👍 
Maybe it would be good to explain that in the comments

Reply:
Definitely, will do in #189